packages/@aws-cdk/app-staging-synthesizer-alpha/test/integ.synth-default-resources.ts

```diff
@@ -17,6 +18,7 @@ const app = new App({
 const stack = new Stack(app, 'synthesize-default-resources', {
   synthesizer: AppStagingSynthesizer.defaultResources({
     appId: APP_ID_MAX, // this has implications on the overall template size
+    stagingBucketEncryption: BucketEncryption.KMS,
```

Comment:
I left this as KMS to avoid changing the integration test. @msambol already added an S3_MANAGED integration test with #28903.